ui/hooks/useTokensToSearch.js

```diff
@@ -129,10 +129,14 @@ export function useTokensToSearch({ usersTokens = [], topTokens = {} }) {
       );
       if (
         isSwapsDefaultTokenSymbol(renderableDataToken.symbol, chainId) ||
-        (usersTokensAddressMap[token.address] &&
-          Number(renderableDataToken.balance ?? 0) !== 0)
```

Comment:
I deleted this line, because I want to include tokens with 0 balance (e.g. a newly imported custom token).

Reply:
However, there is no way to distinguish between a token that had balance at one point and has recently gone to 0, and a token that was just added using the "Add Token" flow. We could add something like a `hasThisTokenEverHadBalance` property to all custom tokens, but that might be over-optimizing at this point.

So perhaps this should be left as it is in this PR for now, but we need to make sure to flag the UX implications to Gal, and this may require improvement in the future.

Reply:
Thanks for all the details, I will check it with product and likely create an improvement user story for this.